order-api/src/main/java/com/smoothstack/order/model/OrderOrderTime.java

```diff
@@ -17,15 +17,15 @@
   @JsonProperty("restaurantAccept")
   private String restaurantAccept;
 
-  @JsonProperty("restaurantStart")
-  private String restaurantStart;
```

Comment:
I'm assuming restaurantAccept will also just be representative of when restaurant starts the order too

Reply:
yeah